js-setup/templates/bootstrapper.html

```diff
@@ -54,15 +54,15 @@
 				'Array.prototype.find|always|gated',
 				'Array.prototype.findIndex|always|gated',
 				'IntersectionObserver'
-			].join(',') + '&excludes=Symbol,Symbol.iterator,Symbol.species,Map,Set');
+			].join(',') + '&excludes=Symbol,Symbol.iterator,Symbol.species,Map,Set' + '&flags=gated&unknown=polyfill');
 		} else {
 			{{#if hasMainCoreJs}}
 			ftNextLoadScript('{{#hashedAsset}}main-core.js{{/hashedAsset}}');
 			{{/if}}
 			ftNextLoadScript(polyfillUrl + '?callback=' + polyfillCallbackName + '&features=' + [
 				'default',
 				'HTMLPictureElement|always|gated'
-			].join(','));
+			].join(',') + '&flags=gated&unknown=polyfill');
```

Comment:
@JakeChampion The polyfill-service is called differently depending on whether the browser "cuts the mustard" or not. The call on line 62 is for ones that don't. After speaking with Rhys we decided that adding `&flags=gated&unknown=polyfill` _shouldn't_ make a difference here just because the browsers _are_ so old by shouldn't be any harm.